### ISSUE TITLE
Vacuum status is not properly shared via MQTT

### ISSUE THREAD

**Issue by tempico** [Hypfer/Valetudo#781]
## Describe the bug
When Valetudo 2021.03.0 is connected to a Home Assistant via MQTT, it shares the incorrect current status. It's set to "Idle" even when robot in reality is cleaning. On the screenshot below you may note the "Idle" status, with the "return to dock" at the same time.

## To Reproduce
1. Setup MQTT integration with Home Assistant auto discovery. Make sure robot in not cleaning (docked)
2. Once robot appears in the entities, check content of `vacuum.rockrobo.attributes.state`
3. Start cleaning
4. Note that status is not updating, whilst the map is re-drawing and other attributes are updating as well.

### Screenshots
<img width="411" alt="Screenshot 2021-03-26 at 13 19 52" src="https://user-images.githubusercontent.com/6486967/112624603-8410be00-8e36-11eb-9f8f-f1dfae0306a0.png">

### Vacuum Model
Roborock S5

### Valetudo Version
Valetudo 2021.03.0 

## Expected behavior
Status attribute shows current status and being updated in a timely manner, allowing to create automations with vacuum (e.g. `if status == 'cleaning': turn on <air_purifier> at 100%`)

**Comment by Hypfer** [comment]
Considering that this is a core functionality used by many users, I'd say you're a bit too fast with deciding that this is a bug in Valetudo and not a quirk of your setup, since otherwise, there would be a lot more reports.

I can't reproduce that.
![image](https://user-images.githubusercontent.com/974410/112627850-6ccfd180-8e32-11eb-8f5c-8cbd6556e9dd.png)

**Comment by tempico** [comment]
Try to build an automation, with attribute I mentioned above.
In the "history" I also have proper status. But not on the status field.
<img width="402" alt="image" src="https://user-images.githubusercontent.com/6486967/112628190-39457500-8e3b-11eb-84ee-4bc1c12e0329.png">

**Comment by Hypfer** [comment]
But why would you look at `vacuum.rockrobo.attributes.state` when the state is the state of the vacuum.rockrobo entity?

Also, all of those attributes don't exist anymore since 2021.01.1 so whatever you're running it's not Valetudo 2021.3.0


This is not a bug but a support request.

**Comment by tempico** [comment]
I was not aware that the attributes got removed. Could be I missed this part in the change logs.
I was able to get the proper status from the `vacuum.rockrobo.state`. Thank you a lot for pointing this. 

However take a look into screenshot below, attributes do update, despite you are saying they don't exist anymore, and the state there is set 3 times: _Template State_ is "docked", _state_ is "idle" and _valetudo_state_ is "idle" as well.
<img width="1502" alt="image" src="https://user-images.githubusercontent.com/6486967/112630265-005acf80-8e3e-11eb-87e0-ec855410a1e5.png">


Valetudo version is latest. I built and re-flashed firmware today via `mirobo`.
![IMG_089FCE2E5502-1](https://user-images.githubusercontent.com/6486967/112630485-4e6fd300-8e3e-11eb-9dae-6c83362cefa9.jpeg)
